app/routes/wow.quantity-manipulation.tsx

```
@@ -0,0 +1,396 @@
+            itemSubClass={parseInt(loaderData.itemSubClass)}
+            onChange={(itemClassValue, itemSubClassValue) => {
+              handleFormChange('itemClass', itemClassValue.toString())
+              handleFormChange('itemSubClass', itemSubClassValue.toString())
+            }}
+          />
+          <RegionAndServerSelect
+            serverSelectFormName="homeRealmName"
+            region={loaderData.region as WoWServerRegion}
+            defaultRealm={
+              {
+                id: parseInt(loaderData.homeRealmName.split('---')[0], 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)